### ISSUE TITLE
Submit Ballot - confirmation window - multiple races

### ISSUE THREAD

**Issue by masiarek** [Equal-Vote/star-server#518]
Link to election: https://dev.star.vote/Election/d93e92a9-d430-4691-9af9-676640b3f725/

1) In general - it is not clear which race is using which voting method - especially in Race 2 and Race 3.
Maybe we can add after each race name: add the Election method name - in brackets (automatically)?

2) Race 2 - HOA - Budget Approval 2024 is using Plurality Voting 
Should we replace all “0” with “No” 
and 
Replace “1” with “Yes” (this is ‘choose-one” race) 
See below in Blue color...

3) Race 3 - Christmas Party - replace “1” with “Approve”
We are using Approval Voting - do we want to replace all “1” (see in red color) with the word “Approve”.
Should we keep “0” as-is or replace it with “Reject / No opinion / Abstain”.

See more details: https://docs.google.com/document/d/113imjTnwrjkySOKDLvtD_In-g3dfCEoVmI0a9CjojMk/edit

<img width="637" alt="image" src="https://github.com/Equal-Vote/star-server/assets/857777/21cf148c-b01c-4559-b1ce-660b677524a4">

**Comment by ArendPeter** [comment]
I agree this view can use some work. We should have the ballots for each race have a design that's more tailored to the voting method that was used instead of showing numbers for all of them
